.github/workflows/python-package.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.8"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r tests/requirements.txt
+    - name: Test with pytest
+      run: |
+        if [[ $TRAVIS_PYTHON_VERSION == pypy* ]]; then pytest -o 'flake8-ignore=*.py ALL'; fi
+        if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then pytest; fi
```

Comment:
We certainly can't refer to TravisCI-specific variables. In this case, I'd try removing the conditional entirely for now, so that we can assess if it's still needed (maybe the issue of flake8 running slowly in pypy is fixed now).

Reply:
Sorry for that, I forgot to take a look at those conditions.
Without the conditions, the workflow for the different 3.x python versions finishes in ~20s, but with pypy it lasts nearly 1m30, so I guess that the issue isn't fixed. I can add the same condition with the github actions syntax.

Reply:
Sure, that sounds good to me. Thanks for checking!